changed tag_map, morph_rules, lemmatizer for Norwegian

katarkor opened the pull request:
<!--- Provide a general summary of your changes in the title. -->

## Description
<!--- Use this section to describe your changes. If your changes required
testing, include information about the testing environment and the tests you
ran. If your test fixes a bug reported in an issue, don't forget to include the
issue number. If your PR is still a work in progress, that's totally fine – just
include a note to let us know. -->

### Types of change
<!-- What type of change does your PR cover? Is it a bug fix, an enhancement
or new feature, or a change to the documentation? -->

## Checklist
<!--- Before you submit the PR, go over this checklist and make sure you can
tick off all the boxes. [] -> [x] -->
- [x] I have submitted the spaCy Contributor Agreement.
- [ ] I ran the tests, and all new and existing tests passed.
- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.

ines commented:
Thanks a lot! (And omg, sorry about our bot – I made a slight fix to its logic earlier and looks like it got stuck in a loop 🙈)

The PR all looks good – there's currently only [one small failure on Travis](https://travis-ci.org/explosion/spaCy/jobs/405307874#L1106) on Python 2 – I think it's complaining about the unicode in the docstring at the top of `lookup.py`. I just moved the `# coding: utf8` up to the top, so this should hopefully fix it.

katarkor commented:
Haha, it's ok :) I updated the contributor agreement now, does it look correct?
>  I think it's complaining about the unicode in the docstring at the top of lookup.py. I just moved the # coding: utf8 up to the top, so this should hopefully fix it.
>
Sorry, didn't think about it. Let's hope it's all good now :)

honnibal commented:
Great work! :man_cartwheeling: 

Especially excited to have more rule-based lemmatizers. I hope this can be used as an example to help lemmatizer rules for Danish and Swedish as well.
